modules/op-resolver-cluster-tasks-ref.adoc

```diff
@@ -0,0 +1,693 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-cluster-tasks-ref_{context}"]
+= Tasks provided in the {pipelines-shortname} namespace
+
+A {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortnames} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
+
+`ClusterTask` functionality is deprecated since version 1.10 and is planned for removal in a future release. If your pipelines use `ClusterTasks`, you can re-create them with the tasks that are available from the {pipelines-shortname} installation namespace by using the cluster resolver. However, certain changes are made in these tasks compared to the existing `ClusterTasks`.
+
+You cannot specify a custom execution image in any of the tasks available in the {pipelines-shortname} installation namespace. These tasks do not support parameters such as `BUILDER_IMAGE`, `gitInitImage`, or `KN_IMAGE`. If you want to use a custom execution image, create a copy of the task and replace the image by editing the copy.
+
+[discrete]
+[id="op-taskref-buildah_{context}"]
+== buildah
+
+The `buildah` task builds a source code tree into a container image and then pushes the image to a container registry.
+
+.Example usage of the `buildah` task
+[source,yaml,subs="attributes+"]
```

Comment:
Not critical, but just wondering if there's a reason to add the attributes sub if you're not replacing anything here?

Reply:
There are replacements in other snippets so I use the same header for uniformity